javascript-practice/src/js/views/dashboardView.js

```diff
@@ -242,24 +245,24 @@ export default class DashboardView {
       }
 
       if (isValid) {
-        if (userId) {
+        if (id) {
           submitUpdateUser({
-            userId,
+            id,
             firstName,
             lastName,
             email,
-            phoneNumber,
+            mobile,
             role,
             userName,
             password,
           });
         } else {
           submitAddUser(
-            userId,
+            id,
```

Comment:
Why add user have id?

Reply:
Because here I have declared a value variable for the id field but when addUser will not allow entering the id. However, when I editUser, I display that id on the form. So I have to define these two functions with the same data types to avoid passing wrong data when I addUser

Reply:
Still can not get it, if you remove id from submitAddUser, what happen here?